rpms/pulp/pulp.spec

```diff
@@ -0,0 +1,853 @@
+
+
+# ---- Admin Client (CLI) ------------------------------------------------------
+%package admin-client
+Summary: Admin tool to administer the pulp server
+Requires: python2-%{name}-common = %{version}
+Requires: python2-%{name}-bindings = %{version}
+Requires: python2-%{name}-client-lib = %{version}
+
+
+%description admin-client
+A tool used to administer the pulp server, such as repository creation and
+synchronization, and to kick off remote actions on consumers.
+
+
+%files admin-client
+%license LICENSE
+%doc README COPYRIGHT
+%doc %{_mandir}/man1/pulp-admin.1*
+%config %{_sysconfdir}/bash_completion.d/pulp-admin
+%dir %{_sysconfdir}/%{name}/admin
+%config(noreplace) %{_sysconfdir}/%{name}/admin/admin.conf
+%dir %{_sysconfdir}/%{name}/admin/conf.d
+%{_bindir}/%{name}-admin
+%dir %{_usr}/lib/%{name}/admin
+%dir %{_usr}/lib/%{name}/admin/extensions
+%{python2_sitelib}/%{name}/client/admin/
+%{python2_sitelib}/pulp_client_admin*.egg-info
+
+
+# ---- Agent -------------------------------------------------------------------
+%package agent
+Summary: The Pulp agent
+Requires: gofer >= %{gofer_version}
+Requires: m2crypto
+Requires: %{name}-consumer-client = %{version}
+Requires: python-gofer >= %{gofer_version}
+Requires: python2-%{name}-bindings = %{version}
+Requires: python2-%{name}-agent-lib = %{version}
+
+
+%description agent
+The pulp agent, used to provide remote command & control and
+scheduled actions such as reporting installed content profiles
+on a defined interval.
+
+
+%files agent
+%license LICENSE
+%doc README COPYRIGHT
+%config(noreplace) %{_sysconfdir}/gofer/plugins/pulpplugin.conf
+%config(noreplace) %{_sysconfdir}/%{name}/agent/agent.conf
+%{python2_sitelib}/%{name}/agent/gofer/
+
+
+# ---- Consumer Client (CLI) ---------------------------------------------------
+%package consumer-client
+Summary: Consumer tool to administer the pulp consumer
+Requires: python2-%{name}-common = %{version}
+Requires: python2-%{name}-bindings = %{version}
+Requires: python2-%{name}-client-lib = %{version}
+
+
+%description consumer-client
+A tool used to administer a pulp consumer.
+
+
+%files consumer-client
+%license LICENSE
+%doc README COPYRIGHT
+%config %{_sysconfdir}/bash_completion.d/pulp-consumer
+%dir %{_sysconfdir}/pki/%{name}/consumer
+%ghost %{_sysconfdir}/pki/%{name}/consumer/rsa_pub.key
+%ghost %dir %{_sysconfdir}/pki/%{name}/consumer/server
+%ghost %{_sysconfdir}/pki/%{name}/consumer/server/rsa_pub.key
+%ghost %{_sysconfdir}/pki/%{name}/consumer/consumer-cert.pem
+%dir %{_sysconfdir}/%{name}/consumer
+%dir %{_sysconfdir}/%{name}/consumer/conf.d
+%config(noreplace) %{_sysconfdir}/%{name}/consumer/consumer.conf
+%{_bindir}/%{name}-consumer
+%dir %{_usr}/lib/%{name}/consumer
+%{python2_sitelib}/%{name}/client/consumer/
+%{python2_sitelib}/pulp_client_consumer*.egg-info
+%doc %{_mandir}/man1/pulp-consumer.1*
+
+%defattr(640,root,root,-)
+%ghost %{_sysconfdir}/pki/%{name}/consumer/rsa.key
+
+%post consumer-client
```

Comment:
Sigh, this hasn't been fixed in Fedora yet and I copied this spec from there. It's fixed here and there now.